frontend/src/pages/CachePage/CachePage.tsx

```diff
@@ -0,0 +1,77 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { getCache } from "../../api/geocaches";
+import { Geocache } from "../../model/Geocache";
+import NavBar from "../../components/NavBar";
+import "./CachePage.scss";
+
+function CachePage() {
+    let { cacheId } = useParams();
+    const [cache, setCache] = useState<Geocache>({
+        referenceCode: "",
+        name: "",
+        placedDate: "",
+        publishedDate: "",
+        type: "",
+        size: "",
+        postedCoordinates: {
+            latitude: 0,
+            longitude: 0
+        },
+        lastVisitedDate: "",
+        isPremiumOnly: false,
+        shortDescription: "",
+        longDescription: "",
+        hints: ""
+    });
+
+    if (cacheId?.at(0) == ":") { cacheId = cacheId.slice(1); }
```

Comment:
I don't understand why this is necessary. The id won't have this symbol. This line can be removed.

Reply:
I've added this line, because id was with ":" in start and caused an error.

Reply:
Ok, I got it now

Reply:
I was not able to reproduce that issue. For me it worked fine without it, for example when the url is localhost:3000/geocaches/zyx. In what situation would there be a ":" symbol?

Reply:
If the issue no longer happens, then that line should be removed.

Reply:
I've got it after last commit, so I change it in the next commit. (I was just giving a link with ":" in start by myself :)

Reply:
Ok 👍